File: 0389. Find the Difference.cpp
// 389.✅ Find the Difference

class Solution
{
public:
    char findTheDifference(string s, string t)
    {
        sort(s.begin(), s.end());
        sort(t.begin(), t.end());
        for (int i = 0; i < s.size(); ++i)
        {
            if (s[i] != t[i])
                return t[i];
        }
        return t[s.size()];
    }
};

File: 0389-find-the-difference/0389-find-the-difference.java
class Solution {
    public char findTheDifference(String s, String t) {
        
        char ch = 0;
        
        for(char c : s.toCharArray())
            ch ^= c;
        
        for(char c : t.toCharArray())
            ch ^= c;
        
        return ch;
    }
}